slate-video-library/src/App.js

```diff
@@ -17,13 +23,59 @@ function App() {
         <Route exact path="/" element={<Homepage />} />
         <Route exact path="/explore" element={<Explorepage />} />
         <Route exact path="/explore/:_id" element={<SingleVideopage />} />
-        <Route exact path="/playlists" element={<Playlistspage />} />
+        <Route
+          exact
+          path="/playlists"
+          element={
+            <Authentication>
+              <Playlistspage />
+            </Authentication>
+          }
+        />
         <Route exact path="/playlists/:_id" element={<PlaylistViewpage />} />

```

Comment:
suggestion: instead of  :_id  use proper naming like playlist_id if you used multiple ids.

Reply:
there is only on id thats _id